package.json

```diff
@@ -40,9 +40,10 @@
     "express": "^4.12.4",
     "express-partial-templates": "^0.1.0",
     "express-session": "^1.11.3",
-    "hof": "^1.2.0",
+    "hof": "^8.0.0",
```

Comment:
I have no idea, we tried bumping it to 7.1 before but broke on node < 3 because of reasons discussed earlier